examples/lm1b/decode.py

```diff
@@ -33,53 +43,87 @@ def multinomial(rng, logits):
   probs = jnp.cumsum(probs, axis=-1)
   a = jax.random.uniform(rng, logits.shape[:-1] + (1,))
   out = jnp.argmin(a > probs, axis=-1)
-  return out[..., None]
+  return out
```

Comment:
multinomial seems useful to upstream to jax.random?

Reply:
probably!  but we'll do that separately then later remove this.